bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -210,6 +210,10 @@ spec:
           resources: {{- toYaml .Values.pgpool.resources | nindent 12 }}
           {{- end }}
           volumeMounts:
+            {{- if or (.Files.Glob "files/pgool.conf") .Values.pgpool.configuration }}
+            - name: pgpool-config
+              mountPath: /opt/bitnami/pgpool/conf/
```

Comment:
Is this the only file that will be written in this folder? Can you confirm that pgpool will never write in this file (I imagine examples like sentinel.conf in redis, that gets written)?

Reply:
I can confirm. See https://github.com/bitnami/bitnami-docker-pgpool#configuration-file